code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -76,7 +76,7 @@
 	gain_text = "Blind man walks through the world, unnoticed by the masses."
 	desc = "Curse someone with 2 minutes of complete blindness by sacrificing a pair of eyes, a screwdriver and a pool of blood, with an object that the victim has touched with their bare hands."
```

Comment:
Curse someone with 2 minutes of complete blindness by sacrificing a pair of eyes and a screwdriver with an object that the victim has touched with their bare hands.